webpack-alias.js

```diff
@@ -1,3 +1,6 @@
+// Needed until https://github.com/benmosher/eslint-plugin-import/issues/1396
```

Comment:
I would recommend us to subscribe to this issue and try it again once closed.

Reply:
:+1: Did so